subsys/caf/events/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+target_sources(app PRIVATE
```

Comment:
Why are we added files to the `app` library ?

The `app` library is the users library, and solely the user should decide what goes in there.

Reply:
That's right. Changed to `zephyr_sources(_ifdef)`,similar to other subsystems like `shell`.